src/platforms/javascript/guides/cordova/ionic.mdx

```diff
@@ -1,11 +1,17 @@
 ---
 title: Ionic
 sidebar_order: 201
+noindex: true
 redirect_from:
   - /clients/cordova/ionic/
 description: "Learn how to use Sentry with Ionic."
 ---
 
+<Alert level="warning" title="Warning">
+
+It is highly recommended to use Ionic with Capacitor, _click here_ to be redirected to the new documentation of Ionic with Sentry Capacitor.
+</Alert>
```

Comment:
```suggestion
We strongly recommended that you use Ionic with Capacitor. Check out our [Ionic with Sentry Capacitor documentation](/platforms/javascript/guides/capacitor/ionic/.

</Alert>
```

Reply:
Oh thanks for the comment, I was looking on how to properly link a page from another framework